src/pages/academy/game/Game.tsx

```diff
@@ -16,10 +16,11 @@ function Game() {
   const [isUsingMock, setIsUsingMock] = React.useState(false);
 
   React.useEffect(() => {
-    createSourceAcademyGame();
+    const game = createSourceAcademyGame();
     return () => {
-      SourceAcademyGame.getInstance().isMounted = false;
-      SourceAcademyGame.getInstance().stopAllSounds();
+      game.isMounted = false;
+      game.stopAllSounds();
+      game.destroy(false);
```

Comment:
This should be `game.destroy(true)`. During testing, I found that `game.destroy(false)` leads to weird behaviour in which the game needs to be loaded at least twice to work. So far, `game.destroy(true)` seems to do the bug fix while not leading to weird behaviour.